### DISCUSSION TITLE
? Post min / max charges for Medicaid and MA Payer Plans

### DISCUSSION THREAD

**Discussion by CarrieLinn** [CMSgov/hospital-price-transparency#76] [Q&A] [answered]
Based on the 6/27/23 FAQ pdf, does anyone know if there is any more recent guidance with respect to considering if Medicaid and MA plans are to have min/max prices published in the MRF?

[hospital-price-transparency-frequently-asked-questions_Updated_06-27-2023.pdf](https://github.com/CMSgov/hospital-price-transparency/files/14875459/hospital-price-transparency-frequently-asked-questions_Updated_06-27-2023.pdf)

**Comment by RidenourK** [comment, marked as answer]
@CarrieLinn Thank you for your question. 

The last two FAQs on page 10 of the FAQs document you linked in your post address this question and are still accurate and applicable.
